module/library/institutionalAssessment/helper.js

```diff
@@ -0,0 +1,97 @@
+/**
+ * name : helper.js
+ * author : Aman
+ * created-date : 04-Jun-2020
+ * Description : Institutional library helper functionality.
+ */
+
+// Dependencies 
+const solutionsHelper = require(MODULES_BASE_PATH + "/solutions/helper");
+
+ /**
+    * InstitutionalHelper
+    * @class
+*/
+module.exports = class InstitutionalHelper {
+
+    /**
+      * List of institutional solution templates.
+      * @method
+      * @name list
+      * @param {String} search - Search Data.
+      * @param {Number} limit - limitting value.
+      * @param {Number} pageNo
+      * @returns {Object} returns list of institutional solutions
+     */
+
+    static list(search,limit,pageNo) {
+        return new Promise(async (resolve, reject) => {
+            try {
+                
+                let institutionalSolution = 
+                await solutionsHelper.templates( 
+                  messageConstants.common.INSTITUTIONAL,
+                  search, 
+                  limit, 
+                  pageNo,  
+                );
+
+                return resolve({
+                  message : messageConstants.apiResponses.INSTITUTIONAL_SOLUTIONS_LIST,
+                  result : institutionalSolution
+                });
+            } catch (error) {
+                return reject({
+                    status: error.status || httpStatusCode.internal_server_error.status,
+                    message: error.message || httpStatusCode.internal_server_error.message,
+                    errorObject: error
+                });
+            }
+        })
+    }
+
+    /**
+      * Institutional solution details information
+      * @method
+      * @name details
+      * @param {String} templateId - Template id.
+      * @returns {Object} returns creator,about and questions per ecm.
+     */
+
+    static details(templateId) {
+        return new Promise(async (resolve, reject) => {
+            try {
+
+              let data = 
```

Comment:
@aman-tunerlabs - Can change the variable name to something else, data sounds too generic, also why is criteriaAndQuestions function name returning details like author, description, link title etc?